harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -762,47 +835,72 @@ def create_lr_scheduler(
             :py:class:`det.pytorch.LRScheduler`:
                 Wrapper around a :obj:`torch.optim.lr_scheduler._LRScheduler`.
         """
-        # TODO(DET-3267): deprecate this when releasing pytorch flexible primitives.
+        # TODO(DET-3262): remove this backward compatibility of old interface.
         pass
 
     @abstractmethod
     def train_batch(
         self, batch: TorchData, model: nn.Module, epoch_idx: int, batch_idx: int
```

Comment:
blocking: `train_batch` must not accept a `model` argument when we make this interface public, since that is a relic of the old API.

Reply:
Removing the `model` is going to break user code. I don't think we should remove this as long as we decide to maintain the backward compatibility.